static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -27,13 +34,14 @@ define([
                 var questionText = question.content;
 
                 if (questionText && atomId) {
-
                     var questionElementClicked = $('#' + questionId);
```

Comment:
`$` returns an array, so your tests will always succeed. I would replace these three lines as follows:
```
var questionElementClicked = document.getElementById(questionId);
var thankYouMessage = document.getElementById('js-question-thankyou-' + questionId);
var signupForm = document.forms['js-storyquestion-email-signup-form-' + questionId];
```
then the test makes sense. Class manipulation will become `element.classList.add` and `element.classList.remove`.